lib/glueby/internal/wallet/tapyrus_core_wallet_adapter.rb

```diff
@@ -28,9 +28,19 @@ def tpc_to_tapyrus(tpc)
         RPC_WALLET_ERROR_ERROR_CODE = -4 # Unspecified problem with wallet (key not found etc.)
         RPC_WALLET_NOT_FOUND_ERROR_CODE = -18 # Invalid wallet specified
 
-        def create_wallet
-          wallet_id = SecureRandom.hex(16)
-          RPC.client.createwallet(wallet_name(wallet_id))
+        def create_wallet(wallet_id = nil)
+          wallet_id = SecureRandom.hex(16) unless wallet_id
+          begin
+            RPC.client.createwallet(wallet_name(wallet_id))
+          rescue RuntimeError => ex
```

Comment:
In latest `tapyrusrb`, `TapyrusCoreClient` throw `Tapyrus::RPC::Error`.

Reply:
fixed at 681ff17